include/zephyr/input/input_touch.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2024 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_TOUCH_H_
```

Comment:
Nit- might want to make the header guard a bit more generic, maybe `ZEPHYR_INCLUDE_INPUT_TOUCH_H_`?

Reply:
Done